README.org

```diff
@@ -236,6 +236,9 @@ variables and functions with their descriptions.
   components.
 - =consult-flymake=: Jump to Flymake diagnostic, like
   =consult-flycheck=.
+- =consult-yasnippet=: Expand [[https://github.com/joaotavora/yasnippet][yasnippet]] snippets in the current buffer.
+  This command is available from the additional =consult-yasnippet.el=
+  package.
```

Comment:
This should probably go to the editing section?

Reply:
Moved.